src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -142,12 +143,22 @@ public Integer getTotalUnReviewedDars(AuthUser authUser) {
      * @param authUser Filter on what DACs the auth user has access to.
      * @return List of DataAccessRequestManage objects
      */
-    public List<DataAccessRequestManage> describeDataAccessRequestManageV2(AuthUser authUser) {
+    public List<DataAccessRequestManage> describeDataAccessRequestManageV2(AuthUser authUser, Optional<String> roleName) {
+        if (roleName.isPresent()) {
+            if (roleName.get().equalsIgnoreCase(UserRoles.SIGNINGOFFICIAL.getRoleName())) {
+                User user = userDAO.findUserByEmailAndRoleId(authUser.getName(), 7);
+                if (Objects.nonNull(user) && Objects.nonNull(user.getInstitutionId())) {
+                    List<DataAccessRequest> dars = dataAccessRequestDAO.findAllDataAccessRequestsForInstitution(user.getInstitutionId());
+                    List<DataAccessRequest> openDars = filterOutCanceledDars(dars);
+                    return createAccessRequestManageV2(openDars);
+                } else {
+                    throw new NotFoundException("Signing Official user associated to an Institution cannot be found.");
+                }
+            }
```

Comment:
I think the flow should be: 

- Check if role is present
- Find auth user
  - throw a 404 if user not found
  - ensure that the user has the role
  - throw a 403 error if user is trying to access a role they don't have
- query on DARs for the user/institution

However, the user validation work should really happen in the resource class, so consider moving the find-user/throw-error logic there so the service class can focus on doing only the work of finding DARs for a role. This would be easier of the service method took a full `User` object instead of an `AuthUser` object.